### DISCUSSION TITLE
Validation of BSNR?

### DISCUSSION THREAD

**Discussion by mge-medisoftware** [Digitaler-Impfnachweis/certification-apis#41] [Q&A] [answered]
The JSON contains the BSNR in the id-Element (in scenario Medical Practitioner office).
Is the BSNR validated in some manner?
Is some validation of the BSNR against the public certificate of the SMCB performed or projected?

This would be a difficult task, because the BSNR is no member of the public certificate so far as i know (can be derived from the contained telematik-ID only in SOME cases, because some issuers disobeyed the related conventions..).

**Comment by oliver-steinbrecher** [comment, marked as answer]
We'd like to add an validation but share you concerns from a technical point of view . I'll provide an update once we have agreed on a way forward.
